app/assets/stylesheets/_billables.scss

```diff
@@ -53,6 +53,19 @@
   }
 }
 
+.unbill{
+     background:none!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 5 spaces<br>Colon after property should be followed by one space